Test/dafny0/ParserOptions.dfy

```diff
@@ -0,0 +1,38 @@
+// RUN: %dafny /compile:0 /print:"%t.print" /dprint:"%t.dprint" "%s" > "%t"
```

Comment:
To compile and run the program as part of the test, use `/compile:3`. (And I think you can drop the `/print` and `/dprint` options, since they are not needed for this test.)

Reply:
Done, though I left compile at 0; the `print` is just to make sure that we're in non-ghost context (compiling and running makes the test much slower)